mitodl/open-discussions
# Move Done/Cancel manage widget buttons into navbar

#### Pre-Flight checklist


#### What are the relevant tickets?
Part of #1653 

#### What's this PR do?
Moves the Done/Cancel manage widget interface into the navbar

#### How should this be manually tested?
Click manage widgets, make some modifications, click Done. You should see the widget changes preserved. If you clicked Cancel instead you should see the changes discarded.

You should also try clicking 'manage widgets' and then clicking 'Done' without making any changes, which is a different code path because the upload is skipped. 

#### Screenshots (if appropriate)
![screenshot from 2019-01-11 16-39-30](https://user-images.githubusercontent.com/863262/51060951-7fe28800-15bf-11e9-9721-139ab368d2e8.png)

@pdpinch How does this look?
